From aed6038dbf7a29599eec93a4afc3090dac0d3788 Mon Sep 17 00:00:00 2001
From: Prarit Bhargava <prarit@redhat.com>
Date: Mon, 14 Jul 2014 13:41:31 -0400
Subject: [x86] mce: Fix "braodcast" typo

Message-id: <1405345303-24468-2-git-send-email-prarit@redhat.com>
Patchwork-id: 85787
O-Subject: [RHEL7.1 PATCH BZ 1119306 01/13] x86, mce: Fix "braodcast" typo
Bugzilla: 1119306
RH-Acked-by: Dean Nelson <dnelson@redhat.com>
RH-Acked-by: Tony Camuso <tcamuso@redhat.com>

Bugzilla: https://bugzilla.redhat.com/show_bug.cgi?id=1119306

commit a90936845da138209aa5dda0c84269f7482aa3cf
Author: Mathias Krause <minipli@googlemail.com>
Date:   Tue Jun 4 20:54:14 2013 +0200

    x86, mce: Fix "braodcast" typo

    Fix the typo in MCJ_IRQ_BRAODCAST.

    Signed-off-by: Mathias Krause <minipli@googlemail.com>
    Signed-off-by: Borislav Petkov <bp@suse.de>

Cc: Dean Nelson <dnelson@redhat.com>
Signed-off-by: Jarod Wilson <jarod@redhat.com>

diff --git a/arch/x86/include/asm/mce.h b/arch/x86/include/asm/mce.h
index f33f9ac..f23c1d6 100644
--- a/arch/x86/include/asm/mce.h
+++ b/arch/x86/include/asm/mce.h
@@ -70,7 +70,7 @@
 #define MCJ_CTX_IRQ  0x2  /* inject context: IRQ */
 #define MCJ_NMI_BROADCAST 0x4  /* do NMI broadcasting */
 #define MCJ_EXCEPTION  0x8  /* raise as exception */
-#define MCJ_IRQ_BRAODCAST 0x10 /* do IRQ broadcasting */
+#define MCJ_IRQ_BROADCAST 0x10 /* do IRQ broadcasting */
 
 #define MCE_OVERFLOW 0  /* bit 0 in flags means overflow */
 
diff --git a/arch/x86/kernel/cpu/mcheck/mce-inject.c b/arch/x86/kernel/cpu/mcheck/mce-inject.c
index ddc72f8..5ac2d1f 100644
--- a/arch/x86/kernel/cpu/mcheck/mce-inject.c
+++ b/arch/x86/kernel/cpu/mcheck/mce-inject.c
@@ -153,7 +153,7 @@ static void raise_mce(struct mce *m)
   return;
 
 #ifdef CONFIG_X86_LOCAL_APIC
- if (m->inject_flags & (MCJ_IRQ_BRAODCAST | MCJ_NMI_BROADCAST)) {
+ if (m->inject_flags & (MCJ_IRQ_BROADCAST | MCJ_NMI_BROADCAST)) {
   unsigned long start;
   int cpu;
 
@@ -167,7 +167,7 @@ static void raise_mce(struct mce *m)
     cpumask_clear_cpu(cpu, mce_inject_cpumask);
   }
   if (!cpumask_empty(mce_inject_cpumask)) {
-   if (m->inject_flags & MCJ_IRQ_BRAODCAST) {
+   if (m->inject_flags & MCJ_IRQ_BROADCAST) {
     /*
      * don't wait because mce_irq_ipi is necessary
      * to be sync with following raise_local
-- 
1.7.1